repair_tests/preview_repair_test.py

```diff
@@ -61,10 +62,12 @@ def test_preview(self):
         result = node1.repair(options=['ks', '--preview'])
         assert "Total estimated streaming" in result.stdout
         assert "Previewed data was in sync" not in result.stdout
+        assert preivew_failure_count(node1) == 1
```

Comment:
Could we also check that the value of the metric is zero after the first preview repair done in line [35](https://github.com/yasar-baigh-kloud-one/cassandra-dtest/blob/84ebb8a662117f0da0a66449a5dae10bf204afaa/repair_tests/preview_repair_test.py#L35)?